draft-group-privacypass-k-check.md

```diff
@@ -308,6 +308,24 @@ In OHTTP, there is exactly one representation for a gateway configuration – th
 Before using the configuration to encrypt a binary HTTP message to the gateway, clients can run
 K-Check with their configured mirrors to ensure that this configuration is correct for the given gateway.
 
+# Integrated Mirrors
+
+As discussed in {{introduction}}, K-Check's design is motivated by a requirement
+in various systems to protect the user's privacy by ensuring that they remain in a
+sufficiently large anonymity set.  K-Check's availability is also important, so
+that users do not lose access to the system.
+
+In many of these systems, including common configurations for Oblivious HTTP and
+Privacy Pass, there is already a party who is necessarily trusted to protect the
+user's privacy in this way, and whose operational availability is already a
+prerequisite for using the system.  In Oblivious HTTP, this is the Relay; in
+Privacy Pass it is typically a transport proxy.
```

Comment:
```suggestion
Privacy Pass it is typically the Attester.
```

Reply:
OK, I've changed this to "... is often the Attester".  ("Typically" seems too strong, given the joint Issuer-Attester and other models.)